feature/account/setup/src/test/kotlin/app/k9mail/feature/account/setup/ui/specialfolders/SpecialFoldersScreenKtTest.kt

```diff
@@ -33,7 +33,7 @@ class SpecialFoldersScreenKtTest : ComposeTest() {
         assertThat(onNextCounter).isEqualTo(0)
         assertThat(onBackCounter).isEqualTo(0)
 
-        viewModel.effect(Effect.NavigateNext)
+        viewModel.effect(Effect.NavigateNext(true))
```

Comment:
Please use a named argument here and in all the other `Effect.NavigateNext` constructor calls in tests.

Reply:
You always have fresh ideas, why not writing them down for everyone to participate, thanks.

Reply:
Some things are hard to write down in a formal rule. But it always comes down to "how easy is this code to read?".

Easy to read: `enableProgressIndicator(true)`
Somewhat verbose, but fine: `enableProgressIndicator(enable = true)`
Really verbose: `enableProgressIndicator(enableProgressIndicator = true)`

Leaves the reader wondering about the parameter: `Effect.NavigateNext(true)`
The reader has a better idea what the argument is about: `Effect.NavigateNext(isManualSetup = true)`